app/services/api_user_token_service.rb

```diff
@@ -0,0 +1,77 @@
+class ApiUserTokenService
+  def initialize(config = nil, args = {})
+    @config = config || YAML.load_file(Rails.root.join("config/api.yml"))
```

Comment:
Did you consider making this the default arg to `config` in the line above? Would this ever be called `nil`?